### ISSUE TITLE
Replace simple DFFT by Chirp Z transform for higher resolution in Time domain processing

### ISSUE THREAD

**Issue by Chems-bitti** [scikit-rf/scikit-rf#1093]
For time domain processing applications, most modern VNAs use Inverse Chirp Z transform instead of the standard DFFT for higher resolution in the time domain over a more restricted time span. it would be nice to integrate this to scikit-rf, probably using the CZT package which is pretty user friendly.
Here's an example using CZT : https://github.com/garrettj403/CZT/blob/main/examples/time-to-freq-to-time-domain-conversion.ipynb

and here is a whitepaper by Copper Mountain Tech explaining the advantages of using this transform (and precautions to take) : https://coppermountaintech.com/wp-content/uploads/2024/04/The-Inverse-Chirp-Z-Transform-for-VNA-Time-Domain-Processing.pdf

**Comment by mhuser** [comment]
Interesting. The results in the paper looks close to what could be achieved using gating and padding, but maybe the computation is more efficient.

Currently there is an ongoing effort #1077 from @fr to make `scikit-rf` compliant with the reference Metas VNA Tools from [here](https://www.metas.ch/metas/en/home/fabe/hochfrequenz/vna-tools.html).

**Comment by FranzForstmayr** [comment]
Yes there's some open work from my side, but I think it's not needed to replace simple dft, as it's used widely in the industry.
czt could be added in addition.
